### DISCUSSION TITLE
How to change the Progress color bar in daisyUI?

### DISCUSSION THREAD

**Discussion by PoohJr** [saadeghi/daisyui#2949] [Q&A]
![PokemonData jsx - PokeApp - Visual Studio Code 4_1_2024 5_53_12 AM](https://github.com/saadeghi/daisyui/assets/145505256/7a0af8d6-9c4b-4062-a42b-d5725f3ae440)

i want to change the bar color depending on the Stat name is that possible?

**Comment by saadeghi** [comment]
You can use conditional class names. For example, if it should be red, add `progress-error` class. If it should be yellow, add `progress-warning` class…

Let me know if you have a question.
